### ISSUE TITLE
Review editors don't reload with browser refresh

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1817]
@waynevicknair 

If you have a review open and hit browser refresh, the review session doesn't reload (which causes other bugs downstream). The problem is at Workbench.js:75 (`if (!path.startWith...`):

```
        for (var i=0;i<state.editors.length;i++) {
            if(singleProject){
                // if running in single user mode, only load editors open for specific projects
                var path = new Path(state.editors[i]);
                if (!path.startsWith(project)) {
                    continue;
                }
            }
```

The review editor is outside the current project, so it doesn't get loaded. My guess is that we need to say "either in current project or in the reviews folder".

**Comment by JonFerraiolo** [comment]
Dupe of #1566 which was fixed. Closing.

**Comment by childsb** [comment]
Fixed this with commit b6d3a8bef3d412ff51fd22165166499e0d7e4af7
